src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,71 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @Test
+    public void splitTestToArrayMultiValue() {
+
+        //given
+        String inputValue = "1,2";
+
+        // when
+        String[] split = inputValue.split(",");
+
+        // then
+        assertThat(split).contains("1", "2");
+        assertThat(split).containsExactly("1", "2");
+    }
+
+    @Test
+    public void splitTestToArraySingleValue() {
+
+        //given
+        String inputValue = "1,";
+
+        // when
+        String[] split = inputValue.split(",");
+
+        // then
+        assertThat(split).contains("1");
+        assertThat(split).containsExactly("1");
```

Comment:
```suggestion
        assertThat("1".split(",")).contains("1");
```

위와 같이 극단적으로 단순하게 구현해도 되지 않을까?